<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>URL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtURL">
        <property name="text">
         <string>http://127.0.0.1:8080</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::MinimumExpanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>User</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtUser">
        <property name="text">
         <string>manager</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="listUsers">
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>614</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionRefresh"/>
    <addaction name="actionNew_User"/>
    <addaction name="actionBlock_User"/>
    <addaction name="actionUnblock_User"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh"/>
   <addaction name="actionNew_User"/>
   <addaction name="actionBlock_User"/>
   <addaction name="actionUnblock_User"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRefresh">
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh List</string>
   </property>
  </action>
  <action name="actionNew_User">
   <property name="text">
    <string>New User</string>
   </property>
   <property name="toolTip">
    <string>New User on database</string>
   </property>
  </action>
  <action name="actionBlock_User">
   <property name="text">
    <string>Block User</string>
   </property>
  </action>
  <action name="actionUnblock_User">
   <property name="text">
    <string>Unblock User</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
